src/codewhisperer/util/runtimeLanguageContext.ts

```diff
@@ -96,6 +97,7 @@ export class RuntimeLanguageContext {
             java: 'java',
             javascript: 'js',
             json: 'json',
+            jsonc: 'json',
```

Comment:
can you check if this works with STE passing json(or it's variant) as the payload and didn't throw a validationException?

Reply:
You mean to write a test case to validate the json or jsonc support to security scans?

Reply:
not only for getDependencyGraph, but also the SendTelemetryEvent with codeScanEvent as the payload.